org.mwc.debrief.core/intro_content/debrief_tutorials.xml

```diff
@@ -4,29 +4,15 @@
 		alt-style="css/overview.properties" style="css/intro.css" name="Debrief bits" path="tutorials/@">
 		<group label="Finding your sea legs in DebriefNG" style-id="content-group"
 			id="orgmwcdebriefcore-introLink-group">
-			<link style-id="content-link" label="A Debrief walkthrough"
-				url="http://org.eclipse.ui.intro/showStandby?partId=org.eclipse.platform.cheatsheet&amp;input=org.mwc.debrief.help.started.tutorial"
+			<link style-id="content-link" label="Debrief tutorial"
+				url="http://org.eclipse.ui.intro/runAction?pluginId=org.mwc.debrief.core&#38;class=org.mwc.debrief.core.actions.OpenDebriefTutorialAction&#38;file=DebriefNGTutorial.pdf"

```

Comment:
@pecko  - do we really need the custom action?  Isn't it possible to just refer to the relative URL of the pdf?  
It looks like the *openBrowser* or *openURL* actions would do the job, as described here:
http://help.eclipse.org/mars/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fextension-points%2FintroContentFileSpec.html